src/types/outputFactory.ts

```diff
@@ -269,6 +269,11 @@ export default class OutputFactory {
     if (batocera) {
       output = output.replace('{batocera}', batocera);
     }
+
+    const jelos = gameConsole.getJelos();
```

Comment:
This is great work! I spot-checked a number of the values and they look right. One last step, could you add some unit tests for this in `outputFactory.test.ts`? There are some other examples in there that you can copy.